metapool/sample_sheet.py

```diff
@@ -377,73 +377,62 @@ def merge(self, sheets):
                 else:
                     pass
 
-    def _remap_table(self, table, strict=True):
-        if self.remapper is None:
-            raise ValueError("sample-sheet does not contain a valid Assay"
-                             " type.")
-
-        # Well_description column is now defined here as the concatenation
-        # of the following columns. If the column existed previously it will
-        # be overwritten, otherwise it will be created here. Alternate versions
-        # of the column name have already been resolved at this point.
-
-        # Note that the amplicon notebook currently generates the same values
-        # for this column. If the functionality in the notebook changes, the
-        # output will continue to be redfined with the current values here.
-        well_description = table['Project Plate'].astype(str) + "." + table[
-            'Sample'].astype(str) + "." + table['Well'].astype(str)
+    def _remap_table(self, table, strict):
```

Comment:
Why remove the strict=True?

Reply:
_remap_table() is just a helper method, called in only one place by _add_data_to_sheet(self, table, sequencer, lanes, assay, strict=True). It's redundant for _remap_table() to have a default value for strict as well, since _add_data_to_sheet() already handles that.